### PR TITLE
bootstrap_jquery Template Tag with Options

### PR DESCRIPTION
**Pull request by FlipperPA**
This P.R. allows the `jquery_url` template tag to return the HTML for the slim or full versions of jQuery. It also allows the developer to pass `falsy`, which returns nothing. The default is to `full`. Tests included.

### CODE REVIEW

File: bootstrap4/templatetags/bootstrap4.py [hunk 1]
```diff
@@ -214,52 +239,59 @@ def bootstrap_css():
...
     Return HTML for jQuery tag.
 
-    Adjust url in settings. If no url is returned, we don't want this
-    statement to return any HTML.
+    Adjust the url dict in settings. If no url is returned, we
+    don't want this statement to return any HTML.
     This is intended behavior.
 
-    Default value: ``None``
+    Default value: ``full``
```
Comment (jieter) [inline bootstrap4/templatetags/bootstrap4.py]: It seems you interpret this as the default value for the first argument and that the docstring was quite out of sync with the current functionality. I'd say this line can be removed entirely, as the default value is documented under **Parameters**.

File: bootstrap4/templatetags/bootstrap4.py [hunk 2]
```diff
@@ -214,52 +239,59 @@ def bootstrap_css():
...
 
     **Usage**::
 
-        {% bootstrap_javascript %}
+        {% bootstrap_jquery %}
 
     **Example**::
 
-        {% bootstrap_javascript jquery=1 %}
+        {% bootstrap_javascript jquery='slim' %}
```
Comment (jieter) [inline bootstrap4/templatetags/bootstrap4.py]: this is an an example for `bootstrap_javascript` in the docstring for `bootstrap_jquery`.

File: bootstrap4/templatetags/bootstrap4.py (hunk 1 above)
Comment (FlipperPA) [reply bootstrap4/templatetags/bootstrap4.py]: You're probably correct, I was just following the pattern / artifact from Bootstrap 3. I'll get that in place today.

File: bootstrap4/templatetags/bootstrap4.py (hunk 2 above)
Comment (FlipperPA) [reply bootstrap4/templatetags/bootstrap4.py]: django-bootstrap3 used these examples in the `docstrings` as the documentation is automatically built from these, right? See here: https://github.com/zostera/django-bootstrap4/blob/master/docs/templatetags.rst Thanks for the feedback. Let me know!

**Comment by jieter** [comment]
@FlipperPA thanks, merging.

**Pull request merged**